src/test/scala/nl.knaw.dans.easy.bagstore/component/BagStoreSpec.scala

```diff
@@ -167,4 +170,26 @@ class BagStoreSpec extends TestSupportFixture
     Files.getPosixFilePermissions(bagInStore.resolve("data")) shouldBe dirPermissions
     Files.getPosixFilePermissions(bagInStore.resolve("data/sub")) shouldBe dirPermissions
   }
+
+  it should "result in a failure when the fetch.txt is invalid" in {
+    val uuid1 = UUID.fromString("00000000-0000-0000-0000-000000000001")
+    val tmpFile = testBagPrunedB.resolve("fetch2.txt").toFile
+    val fetchFilePath = testBagPrunedB.resolve("fetch.txt")
+    val exceptionSuffixMessage = "Bag-id found in fetch.txt can not be found in the bag-store:"
+    val printWriter = new PrintWriter(tmpFile)
+
+    Source
+      .fromFile(fetchFilePath.toFile)
+      .getLines()
+      .map(line => line.replaceAll("01", "10"))
+      .foreach(printWriter.println)
```

Comment:
If tmpFile is a better.File:
```suggestion
      .foreach(tmpFile.write)
```